faker-configs/faker-build-config.json

```diff
@@ -28,6 +28,7 @@
     { "type": "AZUREDEVOPS-RELEASE", "params": { "project": "monitoror", "definition": 423, "status": "RUNNING" } },
     { "type": "GITHUB-CHECKS", "params": { "owner": "monitoror", "repository": "monitoror", "ref": "master" } },
     { "type": "GITHUB-CHECKS", "params": { "owner": "monitoror", "repository": "monitoror", "ref": "master", "status": "ACTION_REQUIRED" } },
-    { "type": "GITHUB-CHECKS", "params": { "owner": "monitoror", "repository": "monitoror", "ref": "master", "status": "RUNNING" } }
+    { "type": "GITHUB-CHECKS", "params": { "owner": "monitoror", "repository": "monitoror", "ref": "master", "status": "RUNNING" } },
+    { "type": "GITHUB-PULLREQUEST", "params": { "owner": "monitoror", "repository": "monitoror", "id": 1337, "branch": "fork:feature-branch", "status": "SUCCESS" } }
```

Comment:
Is it Build or MergeRequest ID?

Reply:
I did not put a title to see the basic rendering in faker. Currently with a tital,  "PR#1337 @" is remove ^^'